source/Geant/cuts/CutConverter.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
```

Comment:
Can you add (to all the headers) the license prefix (see https://github.com/Geant-RnD/GeantExascalePilot/blob/master/source/Geant/processes/Process.hpp) ?

You can also consider adding wording along the line of "Originated in the GeantV project".

Thanks.

Reply:
@pcanal  added the license prefix.